### PR TITLE
Allow under 10 years of residence

### PR DESCRIPTION
**Pull request by alex-solo**
### Description

- *Current tests all pass and the new scenarios provided by Patrick also pass (manually tested).
- The new tests will be incorporated into our scenarios spreadsheet along with the changes to the existing scenarios as a result of the "65+1" task that I will be working on next.

#### List of proposed changes:

-

### What to test for/How to test

-

### Additional Notes

-

### CODE REVIEW

File: components/QuestionsPage/index.tsx [hunk 1]
```diff
@@ -219,6 +219,8 @@ export const QuestionsPage: React.VFC = ({}) => {
     const fields = form.visibleFields.filter((field) =>
       stepKeys.includes(field.key)
     )
+
+    console.log('fields', fields)
```
Comment (MarcoGoC) [inline components/QuestionsPage/index.tsx]: consoleDev instead?

File: utils/api/definitions/schemas.ts [hunk 2]
```diff
@@ -214,13 +212,13 @@ export const RequestSchema = Joi.object({
...
-              ? ValidationErrors.socialCountryUnavailable10
-              : ValidationErrors.yearsInCanadaNotEnough10,
-          })
-        }
+        // if (yearsInCanadaSince18 < 10) {
+        //   return helpers.message({
+        //     custom: value
+        //       ? ValidationErrors.socialCountryUnavailable10
+        //       : ValidationErrors.yearsInCanadaNotEnough10,
+        //   })
```
Comment (MarcoGoC) [inline utils/api/definitions/schemas.ts:220]: you don't get an error for an empty if ??

**Review by MarcoGoC — APPROVED** [review]
I dont understand the changes. i guess this is correct ?

File: components/QuestionsPage/index.tsx (hunk 1 above)
Comment (alex-solo) [reply components/QuestionsPage/index.tsx]: oops missed it

File: utils/api/definitions/schemas.ts (hunk 2 above)
Comment (alex-solo) [reply utils/api/definitions/schemas.ts:220]: no error. Will probably delete though

**Pull request merged**